### PR TITLE
NewFeature: moveTab

### PR DESCRIPTION
**Pull request by philia**
This is a simple feature that i've been using these days, simple key combinations 'm0' to move current active tab to the first place, or 'm1' to the second, up to 'm9' is supported. HTH :)

### CODE REVIEW

**Comment by philia** [comment]
My local keymap for this feature and here is the reason why i unmapped the default 'm' behavior. :)

mapkey("m", 'Move Tab', Normal.moveTab, 1);

File: background.js [hunk 1]
```diff
@@ -342,6 +342,13 @@ Service.registerFrame = function(message, sender, sendResponse) {
 Service.unRegisterFrame = function(message, sender, sendResponse) {
     unRegisterFrame(sender.tab.id);
 };
+Service.moveTab = function(message, sender, sendResponse) {
+    var newPos = parseInt(message.position);
+    var activeTabId = sender.tab.id;
+    if (newPos > -1 && newPos < 10) {
+        chrome.tabs.move(activeTabId, {index: newPos});
+    }
+}
```
Comment (brookhong) [inline background.js]: Why newPos must be less than 10?
And please append `;` at line 351 to make sure that js-beautify will help us to beautify the code.

File: content_scripts.js [hunk 2]
```diff
@@ -1615,6 +1615,11 @@ Normal.insertJS = function(url) {
     s.src = url;
     document.lastElementChild.appendChild(s);
 };
+Normal.moveTab = function(pos) {
+    RUNTIME('moveTab', {
+        position: pos
+    });
+}
```
Comment (brookhong) [inline content_scripts.js]: And please append `;` at line 1622 to make sure that js-beautify will help us to beautify the code.

**Comment by brookhong** [comment]
Nice feature, thanks.

But vim-like-marks works great for those sites you access frequently, and you can even create vim-like mark for [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet).

Reference at https://github.com/brookhong/Surfingkeys#vim-like-marks

File: background.js (hunk 1 above)
Comment (philia) [inline background.js]: Tried double digits followed by leader 'm' key strokes, didn't work for me so i gave up by constrain tabs move between 0...9.

**Pull request merged**